src/DataTable/components/Pagination.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+// Actions
+import { changePage } from '../slices/dataTable'
+
+const Pagination = () => {
+  const dispatch = useDispatch();
+  const { currentPageNumber, totalNumberOfPages, rowsPerPage } = useSelector((state) => ({
```

Comment:
It would be nice to create dedicated selector function for that to not pollute pagination code